docs/api.rst

```diff
@@ -25,6 +25,5 @@
 API Docs
 ========
 
-invenio_deposit
----------------
-
+.. automodule:: invenio_deposit.api
```

Comment:
How should I strucuture the docs?

see https://github.com/inveniosoftware/invenio-deposit/issues/96